js/libs/builder.js

```diff
@@ -686,6 +629,44 @@ var app = new Vue({
 
       return string;
     },
+    initDataSourceProvider: function() {
+      var $vm = this;
+      var dataSourceData = {
+        dataSourceTitle: 'Form data source',
+        dataSourceId: $vm.settings.dataSourceId,
+        appId: Fliplet.Env.get('appId'),
+        default: {
+          name: 'Form data for ' + $vm.settings.displayName,
```

Comment:
@tonytlwu has this name been agreed?

Reply:
Yes. I brought it up in a previous comment. It's been updated based on my suggestions.